### DISCUSSION TITLE
posts/nearby-service/

### DISCUSSION THREAD

**Discussion by giscus** [team-json-delivery/team-json-delivery.github.io#6] [comment]
# posts/nearby-service/

오늘은 스터디원 분들과 가상 면접 사례로 배우는 대규모 시스템 설계 기초 2권 의 주변친구 챕터를 스터디를 하며 나눈 내용을 바탕으로 포스팅을 해보려 해요.
면접에서 주변 친구 서비스를 설계해 보라고 하면 어떻게 할까요?
문제를 이해하고 설계 범위를 확정하자 면접 상황에서는 문제를 이해하고 기능의 스펙을 최소화 하는데 집중해야 합니다.
국내 서비스 수준이 아닌 글로벌 서비스 수준의 설계를 원하는지, 주변 친구의 범위가 어느 정도인지, 채팅과 같은 부가기능을 지원해야 되는지에 대해서도 논의해 보면 좋아요. 면접 상황에서는 가급적 핵심 기능에 집중하고 면접관과 이해도를 맞추도록 해요.

https://team-json-delivery.github.io/posts/nearby-service/

**Comment by zoster0828** [comment]
재밌게 잘 읽었습니다. 😁

**Comment by taeyun1215** [comment]
매번 잘 읽고있습니다. 감사합니다!
